### DISCUSSION TITLE
Discover/Browse Collections

### DISCUSSION THREAD

**Discussion by silentdragon14** [sct/overseerr#2583] [Feature Requests]
I love to be able to "Discover" or browse whole collections.

**Comment by TheCatLady** [comment]
There is no way to do this since TMDb doesn't have an endpoint that returns collections.

**Comment by IllMethods** [comment]
If not browse collections, at least get radarr it to monitor the collection every movie added belongs to? Or have the option to enable that like sonarrs allow partial series requests. I'd be down with that. I go to collections and monitor all in Radarr anyway every once in a while just incase.

When searching for movies in radarr, you can choose monitor movie and collections even if said movie isn't a part of a collection. Surely it would be able to be passed onto radarr this way.
